docs/.gitlintrc.example

```diff
@@ -0,0 +1,10 @@
+preset: default # optional, defaults to "default" preset
+rules:
+	subject-no-trailing-whitespace: 
+		- never # rule condition, is never used
+	body-max-line-length:
+		- always # rule condition, always used for commits from this repo
+		- 100 # rule option, can be any value
```

Comment:
I suppose this is the argument to the rule, and there can be many of them?

Reply:
yeah, if there are multiple arguments then they can be passed as an object/array